docker/docker-compose.connect-popup-test.yml

```diff
@@ -18,7 +18,7 @@ services:
     network_mode: service:trezor-user-env-unix
 
   test-run:
-    image: mcr.microsoft.com/playwright:v1.35.1-focal
+    image: mcr.microsoft.com/playwright:latest
```

Comment:
Are we sure taht we want to have latest? I'm in favor of pinning versions.

Reply:
I agree in general. But playwright gives errors when there is a new version so we would have to be updating it too often. Besides in this is for running tests locally and in CI we are already using latest one with `npx playwright install` command.

Reply:
Well in the CI we are using `image: mcr.microsoft.com/playwright:focal` image and then updating playwright with `npx playwright install` which is very time consuming for running locally every time we run a test.